CMakeLists.txt

```diff
@@ -158,25 +237,55 @@ endif()
 
 
 add_executable(universal-robots)
+
 target_sources(universal-robots
   PRIVATE
     src/viam/ur/module/main.cpp
 )
+
 target_link_libraries(universal-robots
   PRIVATE
     viam-ur
 )
+
 install(
   TARGETS
     universal-robots
 )
 
+
 add_executable(universal-robots-test)
+
 target_sources(universal-robots-test
   PRIVATE
     src/viam/ur/module/test.cpp
 )
+
 target_link_libraries(universal-robots-test
   PRIVATE
     viam-ur
 )
+
+
+
+#
+# Installation and packaging
+#
+
+if (VIAM_UR_DISABLE_APPIMAGE)
```

Comment:
Again in the non-appimage case, we need to know the name of the prefix bindir so we can interpolate it in. So, now meta.json is a generated file from the given template.